e2-studio-star-rx72n-firmware/libs/rx_hcsr04/src/rx_hcsr04_isr.h

```
@@ -0,0 +1,382 @@
+ * The HC-SR04 echo pulse width encodes round-trip time-of-flight. At 20°C,
+ * sound travels at 343 m/s. The round-trip conversion factor is:
+ *
+ *   2 cm / (343 m/s * 100 cm/m) = 58.31 µs/cm ≈ 58 µs/cm
+ *
+ * Usage: distance_cm = echo_time_us / k_hcsr04_us_per_cm
+ *
+ * @note This constant is replicated from rx_hcsr04_timing_t::k_hcsr04_us_per_cm_roundtrip
+ *       as a float for direct use in floating-point distance calculations in code examples.
+ * @since Version 1.2.0
+ */
+static const float k_hcsr04_us_per_cm = 58.0f;
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)